app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadFragment.kt

```diff
@@ -484,11 +490,11 @@ class ThreadFragment : Fragment() {
         toolbarSubject.text = subject
 
         threadSubject.setOnLongClickListener {
-            copyStringToClipboard(subject, R.string.snackbarSubjectCopiedToClipboard, mainViewModel.snackBarManager)
+            context.copyStringToClipboard(subject, R.string.snackbarSubjectCopiedToClipboard, mainViewModel.snackBarManager)
             true
         }
         toolbarSubject.setOnLongClickListener {
-            copyStringToClipboard(subject, R.string.snackbarSubjectCopiedToClipboard, mainViewModel.snackBarManager)
+            context.copyStringToClipboard(subject, R.string.snackbarSubjectCopiedToClipboard, mainViewModel.snackBarManager)
```

Comment:
```suggestion
            requireContext().copyStringToClipboard(subject, R.string.snackbarSubjectCopiedToClipboard, mainViewModel.snackBarManager)
```

Reply:
`context` is not the one from the Fragment but the one from binding thanks to our extensions, yeah it's can be confusing at first